src/backend/env/post.rs

```diff
@@ -91,6 +97,19 @@ impl PartialOrd for Post {
     }
 }
 
+impl Storable for Post {
+    const BOUND: ic_stable_structures::storable::Bound =
+        ic_stable_structures::storable::Bound::Unbounded;
```

Comment:
Why don't we have an upper bound for post size? Wouldn't it be more efficient @ulan?

Reply:
I am afraid due to posts sitting in hot memory first and only eventually are written to the stable memory, the question is how do we even ensure the posts didn't get e.g. too many reactions and can still fit into the bounded size? We don't want to serialize the post on every mutation to only count the seriazliation length, do we?

Reply:
Right, when updating an entry in a stable btreemap do we need to serialize the whole post or just the field we want to update?

Reply:
Whole post.

Reply:
The posts have variable length, right? I thought that's why we don't specify upper bound so we avoid fragmentation.